src/drivers/mpu9250/mpu9250.cpp

```diff
@@ -1669,6 +1679,14 @@ MPU9250::measure()
 	arb.y = _accel_filter_y.apply(y_in_new);
 	arb.z = _accel_filter_z.apply(z_in_new);
 
+	math::Vector<3> aval(x_in_new, y_in_new, z_in_new);
+	math::Vector<3> aval_integrated;
+
+	bool accel_notify = _accel_int.put(arb.timestamp, aval, aval_integrated, arb.integral_dt);
+	arb.x_integral = aval_integrated(0);
+	arb.y_integral = aval_integrated(1);
+	arb.z_integral = aval_integrated(2);
+
```

Comment:
I reviewed it and it is an reference output from put
![image](https://cloud.githubusercontent.com/assets/1945821/11318203/c15894c6-8feb-11e5-84ed-60686baa5a35.png)